src/modules/inventing/parseRecipe.js

```diff
@@ -0,0 +1,28 @@
+import { cdn } from '../system/system';
+import getText from '../common/getText';
+import querySelector from '../common/querySelector';
+import querySelectorArray from '../common/querySelectorArray';
+
+function parseIngredientTable(table) {
+  const [have, need] = getText(querySelector('tr:nth-child(2) td', table)).split('/').map((v) => parseInt(v, 10));
+  return {
+    have,
+    need,
+    id: querySelector('img', table).src.match(/(\d+)\.[A-Za-z]+$/)[1],
+  };
+}
+
+function ingredients() {
+  const ingredientTables = querySelectorArray(
+    `#pCC td[background^="${cdn}ui/inventory/"]`,
+  ).map((td) => td.parentElement.parentElement);
+  return ingredientTables.map(parseIngredientTable);
+}
+
+export function findMaxInv() {
```

Comment:
```suggestion
export default function parseRecipe() {
```
either that or rename the file findMaxInv
but definitely use a default export